cashu/wallet/__api__.pi.py

```diff
@@ -0,0 +1,6 @@
+from cashu.wallet.api import app as wallet_api
```

Comment:
This is a weird file name. Please revert and add to `__init__.py`?

Reply:
Do we want to start the `api` as soon as we import the `wallet` ? 
My thought of having it in a separate file, is that you may want to import the wallet without starting the `fastapi` app.
Though i agree, the filename is weird. I'll change to `__api__.py` for now. But definitely open to putting it in `__init__.py` just sharing a few thougths.